### ISSUE TITLE
orgPackages deprecated in emacs-packages

### ISSUE THREAD

**Issue by JonathanLorimer** [nix-community/nix-doom-emacs#27]
This commit https://github.com/NixOS/nixpkgs/commit/932ab304f0b8e3241c1311b9b731d3d330291715#diff-56c122d979843ece29fff08e6d83c47ccf8cdbfedb68e4de21dd0cae5337dcf4 deprecates orgPackages.

This is leading to this error when I try to use `nix-doom-emacs`

```
error: anonymous function at /nix/store/pwz0d92vl5hjrvhiri3qzazxjnf1r8wy-source/pkgs/top-level/emacs-packages.nix:58:53 called with unexpected argument 'orgPackages'

       at /nix/store/pwz0d92vl5hjrvhiri3qzazxjnf1r8wy-source/lib/customisation.nix:69:16:

           68|     let
           69|       result = f origArgs;
             |                ^
           70|
(use '--show-trace' to show detailed location information)
```

**Comment by ckiee** [comment]
Going to handle debugging in #28 since it's probably related.

**Comment by JonathanLorimer** [comment]
Closing as resolved / related to #28
